app/api/utility.py

```diff
@@ -205,3 +205,28 @@ def create_query(
         """
 
     return "\n".join([DEFAULT_CONTEXT, query_string])
+
+
+def create_terms_query(attribute_URI: str) -> str:
+    """
+    Creates a SPARQL query using a simple query template to retrieve term URLS for a given attribute.
+
+    Parameters
```

Comment:
I think this docstring could benefit from an example: https://numpydoc.readthedocs.io/en/latest/format.html#examples so we can immediately understand what we mean by `dataElementURI` or similar